package.json

```diff
@@ -13,6 +13,7 @@
     "author": "Jason Morita",
```

Comment:
This should be `Globality Engineering` now.... The email should be `engineering@globality.com`